.github/workflows/drift-check.yml

```diff
@@ -33,7 +33,15 @@ jobs:
           workload_identity_provider: ${{ secrets.WORKLOAD_IDENTITY_PROVIDER }}
           service_account: ${{ secrets.GOOGLE_SERVICE_ACCOUNT }}
 
-      - uses: simenandre/setup-gke-gcloud-auth-plugin@v1
+      - uses: 'google-github-actions/setup-gcloud@v2'
+        with:
+          install_components: 'gke-gcloud-auth-plugin'
```

Comment:
Maybe … move this before the `google-github-actions/auth` step? Not sure if it matters, but it felt reasonable.

Reply:
But that diverges from the [migration guide](https://github.com/simenandre/setup-gke-gcloud-auth-plugin/pull/8) :O